test/features/UniversityTest.java

```diff
@@ -314,7 +315,6 @@ public void doubleQuotientTest2() {
 
         assertFalse(refines);
     }
-
```

Comment:
Now there seems to be inconsistent use of the empty line between tests. Most of the times there is an empty line between test, but not here (and for example also at lines 292-293).